core/foundation/src/TClassEdit.cxx

```diff
@@ -621,6 +621,10 @@ static size_t findNameEnd(const std::string &full, size_t pos)
 bool TClassEdit::IsDefAlloc(const char *allocname, const char *classname)
 {
    string_view a( allocname );
+   const static int clalloclen = strlen("class ");
```

Comment:
```suggestion
   constexpr static int clalloclen = strlen("class ");
```

Reply:
I've now tried with an alternative function, let's see if it works.

Reply:
Seems solved now